registry/quilt_server/views.py

```diff
@@ -670,8 +683,8 @@ def _generate_preview(node, max_depth=PREVIEW_MAX_DEPTH):
 @app.route('/api/package_preview/<owner>/<package_name>/<package_hash>', methods=['GET'])
 @api(require_login=False)
 @as_json
-def package_preview(auth_user, owner, package_name, package_hash):
-    instance = _get_instance(auth_user, owner, package_name, package_hash)
+def package_preview(owner, package_name, package_hash):
+    instance = _get_instance(g.auth, owner, package_name, package_hash)
```

Comment:
you don't mean `g.auth.user`?

Reply:
The point is to pass all of auth - which will eventually include team info, admin status, etc.